builtin/docker/builder.go

```diff
@@ -265,6 +274,18 @@ func (b *Builder) Build(
 	return result, nil
 }
 
+// Translates BuildArgs from a map of key vals into a string of '--build-arg key=vals'
+func createBuildArgsString(m map[string]*string) string {
+	if m == nil {
+		return ""
+	}
```

Comment:
```suggestion
```

Reply:
Sorry about this! I was wrong yesterday on this feedback. Gonna add a quick suggested change and commit it for simplicity, otherwise this PR looks good to go ❤️ 👍🏻